modules/candidate_list/jsx/openProfileForm.js

```diff
@@ -89,34 +91,32 @@ class AccessProfilePanel extends Component {
               </div>
       );
     }
-    return (<div className='col-sm-3'>
-            <Panel title='Open Profile'>
-            <FormElement
-            name='openprofile'
-                onSubmit={this.validateAndSubmit}
-                onUserInput={this.validateAndSubmit}>
-                <TextboxElement
-                    name='CandID'
-                    label='CandID'
-                    value={this.state.CandID}
-                    onUserInput={this.updateFormElement}
-                 />
-                <TextboxElement
-                    name='PSCID'
-                    label='PSCID'
-                    value={this.state.PSCID}
-                    onUserInput={this.updateFormElement}
-                 />
-                {warning}
-                 <ButtonElement
-                    name='Open Profile'
-                    label='Open Profile'
-                    onUserInput={this.validateAndSubmit}
-                 />
-             </FormElement>
-        </Panel>
-        </div>
-       );
+    return (
+      <FormElement
+        name='openprofile'
+        onSubmit={this.validateAndSubmit}
+        onUserInput={this.validateAndSubmit}>
```

Comment:
I'm not sure if `FormElement` takes a prop called `onUserInput`. Is there are reason for having it here? I also noticed you have the button element at the bottom of this form also triggering `this.validateAndSubmit`. I think believe only one of these is necessary.